public/js/comment.js

```diff
@@ -0,0 +1,26 @@
+const createCommentHandler = async (event) => {
+  event.preventDefault();
+
+  //   Eingabefelder habe ich so ausgelesen
+  const comment = document.getElementById("input-comment").value;
+  const postId = document.getElementById("post-id").textContent;
```

Comment:
great job! :-)
If you can, consider writing your code using jQuery to shorten it a little and make it even more readable e.g.
```
$("#post-id").textContent
```
in place of
```
document.getElementById("post-id").textContent
```